tidb-cloud/release-notes-2022.md

```diff
@@ -8,6 +8,22 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## September 27, 2022
+
+**General changes**
+
+* Support joining multiple organizations by invitation.
+
+    In the console, you can view all organizations you have joined and switch them. For more information, see [Switch organizations](/tidb-cloud/manage-user-access.md#switch-organizations).
+
+* Add the [Slow Query](/tidb-cloud/tune-performance.md#slow-queries) page for SQL diagnosis.
+
+    On the Slow Query page, you can search and view all slow queries in your TiDB cluster, and explore the bottlenecks of each slow query by viewing its [execution plan](https://docs.pingcap.com/tidbcloud/explain-overview), SQL execution information, and other details.
+
+* When you reset the password for your account, TiDB Cloud will check your new password input against your last four passwords, and remind you to avoid using one of them. Any of the four used passwords will not be permitted.
```

Comment:
```suggestion
* When you reset the password for your account, TiDB Cloud will check your new password input against your last four passwords, and remind you to avoid using any of them. Any of the four used passwords will not be permitted.
```